org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurConsole.java

```diff
@@ -64,21 +68,26 @@
  */
 
 public final class OurConsole extends JScrollPane {
+	final static int menuShortcutKeyMask = Toolkit.getDefaultToolkit().getMenuShortcutKeyMask();
 
 	/** This ensures the class can be serialized reliably. */
-	private static final long	serialVersionUID	= 0;
+	private static final long			serialVersionUID	= 0;
 
 	/** The style for default text. */
-	private final AttributeSet	plain				= style("Verdana", 14, false, Color.BLACK, 0);
+	private final static AttributeSet	plain				= style("Verdana", 14, false, false, false, Color.BLACK, 0);
+
+	/** The style for table text. */
+	private final static AttributeSet	mono				= style("Courier", 14, false, false, false, Color.BLACK,
+			10);
 
 	/** The style for bold text. */
-	private final AttributeSet	bold				= style("Verdana", 14, true, Color.BLACK, 0);
+	private final static AttributeSet	bold				= style("Verdana", 14, true, false, false, Color.BLACK, 0);
 
 	/** The style for successful result. */
-	private final AttributeSet	good				= style("Verdana", 14, false, Color.BLUE, 15);
+	private final static AttributeSet	good				= style("Verdana", 14, false, false, false, Color.BLUE, 10);
 
 	/** The style for failed result. */
-	private final AttributeSet	bad					= style("Verdana", 14, false, Color.RED, 15);
+	private final static AttributeSet	bad					= style("Verdana", 14, false, false, false, Color.RED, 10);
```

Comment:
Tabs are evil: at least here on GitHub, the RHS expressions for `good` and `bad` are not aligned.